cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/vis-augmenter/dashboard_spec.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  INDEX_PATTERN_FILEPATH_SIMPLE,
+  INDEX_SETTINGS_FILEPATH_SIMPLE,
+  SAMPLE_DATA_FILEPATH_SIMPLE,
+} from '../../../../../utils/constants';
+import {
+  deleteVisAugmenterData,
+  bootstrapDashboard,
+} from '../../../../../utils/dashboards/vis-augmenter/helpers';
+
+describe('Vis augmenter - existing dashboards work as expected', () => {
+  describe('dashboard with ineligible, eligible, and vega visualizations', () => {
+    const indexName = 'vis-augmenter-sample-index';
+    const indexPatternName = 'vis-augmenter-sample-*';
+    const dashboardName = 'Vis Augmenter Dashboard';
+    const visualizationSpecs = [
+      {
+        name: 'count-agg-vis',
+        type: 'line',
+        indexPattern: indexPatternName,
+        metrics: [],
+      },
+      {
+        name: 'single-metric-vis',
+        type: 'line',
+        indexPattern: indexPatternName,
+        metrics: [
+          {
+            aggregation: 'Average',
+            field: 'value1',
+          },
+        ],
+      },
+      {
+        name: 'multi-metric-vis',
+        type: 'line',
+        indexPattern: indexPatternName,
+        metrics: [
+          {
+            aggregation: 'Average',
+            field: 'value1',
+          },
+          {
+            aggregation: 'Average',
+            field: 'value2',
+          },
+          {
+            aggregation: 'Max',
+            field: 'value3',
+          },
+        ],
+      },
+      {
+        name: 'area-vis',
+        type: 'area',
+        indexPattern: indexPatternName,
+        metrics: [
+          {
+            aggregation: 'Max',
+            field: 'value2',
+          },
+        ],
+      },
+      {
+        name: 'vega-vis',
+        type: 'vega',
+        indexPattern: indexPatternName,
+        metrics: [],
+      },
+    ];
+
+    const visualizationNames = visualizationSpecs.map(
+      (visualizationSpec) => visualizationSpec.name
+    );
+
+    before(() => {
+      // Create a dashboard and add some visualizations
+      bootstrapDashboard(
+        INDEX_SETTINGS_FILEPATH_SIMPLE,
+        INDEX_PATTERN_FILEPATH_SIMPLE,
+        SAMPLE_DATA_FILEPATH_SIMPLE,
+        indexName,
+        indexPatternName,
+        dashboardName,
+        visualizationSpecs
+      );
+    });
+
+    beforeEach(() => {
+      cy.visitDashboard(dashboardName);
+      cy.wait(5000);
+    });
+
+    after(() => {
+      deleteVisAugmenterData(
+        indexName,
+        indexPatternName,
+        visualizationNames,
+        dashboardName
+      );
+    });
+
+    it('All visualizations should be visible', () => {
+      visualizationNames.forEach((visualizationName) =>
+        cy.getVisPanelByTitle(visualizationName)
```

Comment:
Do we need to do a should exist check?

Reply:
Fair enough - this is covered automatically in below tests. I'll remove this one.